raiden-dapp/src/services/raiden-service.ts

```diff
@@ -708,8 +708,14 @@ export default class RaidenService {
     await this.raiden.monitorToken(address);
   }
 
-  async getTokenBalance(tokenAddress: string): Promise<string> {
-    const tokenBalance = await this.raiden.getTokenBalance(tokenAddress);
+  async getTokenBalance(
+    tokenAddress: string,
+    raidenAccount?: string
+  ): Promise<string> {
+    const tokenBalance = await this.raiden.getTokenBalance(
+      tokenAddress,
+      raidenAccount
+    );
     return tokenBalance.toString();
```

Comment:
You sure you want to convert it `toString()`? You need to convert it right after to BigNumber again just to compare.. maybe it'd be better to leave it as a BigNumber, and then to convert only where you need to use it as a string

Reply:
It was being converted to a string initially, I didn't change that. But I can check if not converting it breaks anything, if not I can make it a BigNumber.

Reply:
Yeah, it breaks something in the transfer route component. I'll take a look at that after I've fixed all broken tests for withdrawal.

Reply:
Returning it as a `BigNumber` now.